test/functional/p2p_segwit.py

```diff
@@ -833,6 +832,36 @@ def test_block_malleability(self):
 
         assert self.nodes[0].getbestblockhash() == block.hash_hex
 
+        # Build a relayable-but-invalid block
+        relayable_block = self.build_next_block()
+        add_witness_commitment(relayable_block)
+        relayable_block.solve()
+
+        # Append extra witness data to the coinbase input that triggers the
+        # same validation rejection but keeps the block under the weight limit
+        # so it can be sent via P2P.
+        relayable_block.vtx[0].wit.vtxinwit[0].scriptWitness.stack.append(b'a' * 100_000)
+
+        # Ensure it's relayable by weight
+        assert relayable_block.get_weight() <= MAX_BLOCK_WEIGHT
```

Comment:
19c5dba8ca8ff66f3a8d4a62e2145c67e81e6a6f: you could use `assert_greater_than_or_equal`.